source/modules/style-guide/style-guide.pug

```diff
@@ -0,0 +1,73 @@
+mixin style-guide(projectName, version, date)
+  div.style-guide.container.d-flex.flex-column
+
+    h1.style-guide__heading Style Guide
+      span.style-guide__version Version #{version}
+      span.style-guide__date Update on #{date}
+    p.style-guide__sub-heading
+      | This is a style guideline for
+      span.style-guide__project-name= projectName
+      |. The template includes colours, typography, icons, buttons, forms…
+
+    ol.style-guide__list
+      li.style-guide__list-item
+        span.guide__title Changelog
+        block
+        div.changelog
+          - blocks.changelog()
+      li.style-guide__list-item
+        span.guide__title Color Palette
+        block
+        div.colors
+          - blocks.colors()
+      li.style-guide__list-item
+        span.guide__title Typography
+        block
+        div.typo
+          - blocks.typo()
+      li.style-guide__list-item
+        span.guide__title Icons
+        block
+        div.icons
+          - blocks.icons()
+      li.style-guide__list-item
+        span.guide__title Buttons
+        block
+        div.buttons
+          - blocks.buttons()
+      li.style-guide__list-item
+        span.guide__title Forms
+        block
+        div.forms
+          - blocks.forms()
+      li.style-guide__list-item
+        span.guide__title Updating
+
+mixin colorPalette(color, hexCode)
+  div.card
+    div.card-img-top(style=`background-color: ${hexCode}; height: 10rem;`)
+    div.card-body
+      h5.card-title= color
+      p.card-text= hexCode
+
+mixin typoGuide(fontFamily, fontSize, fontWeight, color)
+    ul.my-2.guide-content__typo
+      li.font-weight-bold Font-family:
```

Comment:
please using data json. after that using loop

Reply:
Updated with data.json and loop